topics/go/algorithms/strings/reverse/reverse.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+
+	fmt.Println(ReverseString("Hello World"))
+}
+
+func ReverseString(str string) string {
+
+	// Here we create a slice of runes.
+	str_table := []rune(str)
+
+	// Here we create int that will be a pointer to the front of the str_table.
+	frontIndex := 0
+
+	// Here  we create int that will be a pointer to the end of the str_table.
+	BackIndex := len(str_table) - 1
```

Comment:
Why use an exported variable here?

Reply:
Instead of `Index` can we use the word `cp` or `codePoint` as in `frontCodePoint` or `fcp` and `bcp` or something with code point in it?

Reply:
It was exported because of a typo.